src/js/components/SkipLinks/SkipLinks.js

```diff
@@ -23,8 +23,8 @@ class SkipLinks extends Component {
       const layerNode = this.layerRef.current;
       if (
         layerNode &&
-        layerNode.contains &&
-        !layerNode.contains(document.activeElement)
+        layerNode.layerContainer.contains &&
```

Comment:
Why we are including this extra level here?

Reply:
@alansouzati I mentioned it on the PR details above:
>...In addition, `layerNode` does not directly contain the `document.activeElement`, so that was fixed as well.
this is due to the change of using `this.layerRef.current` to initialize `layerNode`